subprojects/security/src/testFixtures/groovy/org/gradle/security/fixtures/SigningFixtures.groovy

```diff
@@ -189,13 +195,27 @@ class SigningFixtures {
         signatureSubpacketGenerator.setPreferredSymmetricAlgorithms(false, PublicKeyAlgorithmTags.DSA)
         signatureSubpacketGenerator.setPreferredHashAlgorithms(false, HashAlgorithmTags.SHA512)
         signatureSubpacketGenerator.setPreferredCompressionAlgorithms(false, CompressionAlgorithmTags.ZIP)
+        signatureSubpacketGenerator.addSignerUserID(false, "www")
+
+        PGPUserAttributeSubpacketVectorGenerator userAttributes = new PGPUserAttributeSubpacketVectorGenerator()
+        userAttributes.setImageAttribute(ImageAttribute.JPEG, new byte[100]);
+        PGPUserAttributeSubpacketVector userAttributesVector = userAttributes.generate()
```

Comment:
:grey_question:  Huh, is it neccessary to generate an image attribute, if you want to make an UID subpacket?

Reply:
Nope, but I added it to be sure it would not affect tests in any way and would be removed from the result file